bluemira/geometry/parameterisations.py

```diff
@@ -648,26 +652,36 @@ class TripleArcOptVaribles(OptVariablesFrame):
     sl: OptVariable = ov(
         "sl", 6.428, lower_bound=5, upper_bound=10, description="Straight length"
     )
+    # TODO @CoronelBuendia: can we rename the radii f1 f2 to something like r1 r2?
```

Comment:
This is your todo :)
```suggestion
    # TODO @OceanNuclear: can we rename the radii f1 f2 to something like r1 r2?
```

Reply:
I would suggest that too, "f" for radius is unconventional